segmenters/nlp/PDFPlumberSegmenter/tests/test_pdfextractorcrafter.py

```diff
@@ -0,0 +1,96 @@
+import os
+import pytest
+
+import numpy as np
+from PIL import Image
+
+from .. import PDFPlumberSegmenter
+
+expected_text = "A cat poem\nI love cats, I love every kind of cat,\nI just wanna hug all of them, but I can't," \
+                "\nI'm thinking about cats again\nI think about how cute they are\nAnd their whiskers and their " \
+                "nose"
+
+cur_dir = os.path.dirname(os.path.abspath(__file__))
+path_img_text = os.path.join(cur_dir, 'cats_are_awesome.pdf')
+path_text = os.path.join(cur_dir, 'cats_are_awesome_text.pdf')
+path_img = os.path.join(cur_dir, 'cats_are_awesome_img.pdf')
+
+with open(path_text, 'rb') as pdf:
+    input_bytes_text = pdf.read()
+
+with open(path_img, 'rb') as pdf:
+    input_bytes_image = open(path_img, 'rb').read()
+
+with open(path_img_text, 'rb') as pdf:
+    input_bytes_images_text = pdf.read()
```

Comment:
This code should go into a fixture and not on module-level.

Reply:
Moved to fixture